invalid operation: division by zero

问题描述:
going@CTU-WKS-AA578:~/workspace/test/basic$ go run main.go
# command-line-arguments
./main.go:8:13: invalid operation: division by zero

步骤重现
要重现问题，请按照以下步骤操作：
步骤 1
执行main.go
步骤 2
步骤 3

期望结果
打印出res

实际结果
./main.go:8:13: invalid operation: division by zero

附件
附加任何相关文件或截图。

--- a/main.go
+++ b/main.go
@@ -5,6 +5,6 @@ import (
 )
 
 func main() {
-	res := 1 / 0
+	res := 1 / 1
 	fmt.Println("res: ", res)
 }

